lib/puppet/type/concat_file.rb

```diff
@@ -85,40 +92,56 @@ def exists?
   end
 
   # Inherit File parameters
-  newparam(:selinux_ignore_defaults) do
-  end
+  newparam(:selinux_ignore_defaults, :boolean => true, :parent => Puppet::Parameter::Boolean)
```

Comment:
I removed the parent, then tests failed. It apparently accepted lots of truthy values because of `:boolean => true`, but didn't sanitize them on the way back. I didn't track down as to why yet though. It could have just been poorly defined behavior checking in the unit tests.

Oh how I wish we had actual manifest/catalog type & provider unit testing. Someday.